generate_R_table.py

```diff
@@ -68,8 +69,13 @@ def main():
     # answers to all the questions
     # Key = item number;
     # value = (correct_button_code, LeftorRight)
-    answer_key = dict_from_table(read_table(file_names['Question key filename']),
-                                                    paired=False)
+    
+    # THE TWO LINES BELOW ARE COMMENTED BECAUSE THEY DON'T WORK CORRECTLY.
+    # THE LINE BELOW THEM IS A QUICK FIX (ALSO SEE LINE 396). 
+    #answer_key = dict_from_table(read_table(file_names['Question key filename']),
+    #                                               paired=False)
```

Comment:
Which debugger are you using? According to [this](http://stackoverflow.com/a/9051196/4501212), PDB should step through generators!